os/windows/AudioOutputWASAPI.cpp

```diff
@@ -216,8 +216,20 @@ void AudioOutputWASAPI::ActuallySetCurrentDevice(std::string deviceID){
 
 	if(deviceID=="default"){
 		isDefaultDevice=true;
-		res=enumerator->GetDefaultAudioEndpoint(eRender, eCommunications, &device);
+		IMMDevice* temp_device;
```

Comment:
looks strange since all other variables are in camelCase

Reply:
Y